.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -47,3 +47,8 @@ To be verified by following cases:
 - [ ] 1. Pass/Fail (your name, SW version, date)
 - [ ] 2. Pass/Fail (your name, SW version, date)
 
+## Traceability
+
+_instruction: Check that the pull request is linked to an issue, if not then link it_
```

Comment:
Shouldn't the instruction be the other way around, asking to check that the issue is linked to at least one pull request?

Reply:
yes, seems to be the older instructions were left unchanged, I have now fixed the wording.